src/parse/parser.rs

```diff
@@ -416,7 +413,11 @@ where
 {
     // The actual parsing function
     #[allow(clippy::cognitive_complexity)]
-    pub fn get_matches_with(&mut self, matcher: &mut ArgMatcher, it: &mut Input) -> ClapResult<()> {
+    pub(crate) fn get_matches_with(
```

Comment:
You made the `Parser` `pub(crate)` in `mod.rs`. So, you need to define `crate` here.

Reply:
Define `crate`? Explain?

Reply:
Sorry I meant "You don't need to use `(crate)` here"

Reply:
I'd rather keep things explicit. Does this function have to be exposed to public? If not, it's `pub(crate)` or no `pub` at all.